src/frameworks/angular/index.ts

```diff
@@ -243,19 +243,24 @@ exports.handle = function(req,res) {
 };\n`;
   } else if (serverOutputPath) {
     bootstrapScript = `
-    const app = new Promise((resolve) => {
+    const app = new Promise((resolve, reject) => {
       setTimeout(() => {
```

Comment:
JFYI: The setTimeout here seems to be quite brittle, and I doubt it is longer needed.